BlazarUI/app/scripts/models/Log.js

```diff
@@ -14,8 +13,27 @@ class Log extends Model {
   }
 
   url() {
-    return this.logUrl;
+    return `${config.apiRoot}/build/${this.buildNumber}/log/`;
   }
+
+  formatLog(jqxhr) {
+    let cache = [];
+
+    if (jqxhr.status !== 200) {
+      console.warn(jqxhr);
+      return "<p class='x-y-roomy'>Error loading build log. Please check your console for more detail.</p>";
+    }
+
+    const lines = jqxhr.responseJSON.data.split('\n');
+
+    lines.forEach( (line) => {
+      cache.push(`<p class='log-line'>${line}</p>`);
+    });
+
+    return cache.join('\n');
```

Comment:
On second thought, I have a question about this -- why the newline character between `p` tags? `p` elements are `display: block` so they should render on newlines naturally. I don't think that the `\n` is actually doing anything. Is there another use of this output that I may not be considering? I'm assuming it's only being used as inner html.